### PR TITLE
OIE embedded SDK-SIW guide for Java -  edit review

### PR DESCRIPTION
**Pull request by vanngo-okta**
## Description:
- **What's changed?** Final edit of Java content in the OIE embedded auth guides, and some common content (that affects Java)
- **Is this PR related to a Monolith release?** No

### Resolves:

* [OKTA-408182](https://oktainc.atlassian.net/browse/OKTA-408182)

### CODE REVIEW

File: packages/@okta/vuepress-site/docs/guides/oie-embedded-common-download-setup-app/main/java/clientsecret.md [hunk 1]
```diff
@@ -1,6 +1,7 @@
 ### Client secret
 
-For the sample app, use the **Client secret** for the application that you created in
-[Create your Okta org](/docs/guides/oie-embedded-common-org-setup/java/main/).
-To find this value, go to **Applications** > **Applications** in the Admin Console.
-Select your app, and then on the **General** tab copy the **Client secret**.
+For the sample app, use the **Client secret** for the app integration that you've created in
```
Comment (christinechoy-okta) [inline packages/@okta/vuepress-site/docs/guides/oie-embedded-common-download-setup-app/main/java/clientsecret.md:3]:
```suggestion
For the sample app, use the **Client secret** for the app integration that you created in
```

File: packages/@okta/vuepress-site/docs/guides/oie-embedded-sdk-use-cases/oie-embedded-sdk-use-case-basic-sign-in/java/integrationsteps.md [hunk 2]
```diff
@@ -29,7 +29,7 @@ Depending on [AuthenticationResponse](https://github.com/okta/okta-idx-java/blob
 
 #### Success status
 
-For a successful user sign in (`AuthenticationStatus.Success`), use [AuthenticationResponse](https://github.com/okta/okta-idx-java/blob/master/api/src/main/java/com/okta/idx/sdk/api/response/AuthenticationResponse.java)'s `getTokenResponse` method to retrieve the token and proceed to process the authenticated user request.
+For a successful sign-in response (`AuthenticationStatus.Success`), use [AuthenticationResponse](https://github.com/okta/okta-idx-java/blob/master/api/src/main/java/com/okta/idx/sdk/api/response/AuthenticationResponse.java)'s `getTokenResponse` method to retrieve the token and proceed to process the authenticated user in the app.
```
Comment (christinechoy-okta) [inline packages/@okta/vuepress-site/docs/guides/oie-embedded-sdk-use-cases/oie-embedded-sdk-use-case-basic-sign-in/java/integrationsteps.md:32]:
```suggestion
For a successful sign-in response (`AuthenticationStatus.Success`), use the [AuthenticationResponse](https://github.com/okta/okta-idx-java/blob/master/api/src/main/java/com/okta/idx/sdk/api/response/AuthenticationResponse.java)'s `getTokenResponse` method to retrieve the token and then process the authenticated user in the app.
```

File: packages/@okta/vuepress-site/docs/guides/oie-embedded-sdk-use-cases/oie-embedded-sdk-use-case-basic-sign-in/java/integrationsteps.md [hunk 3]
```diff
@@ -85,5 +85,4 @@ For example:
 
 ### Step 4: Get user profile information-optional
 
-Optionally, you can obtain basic user information after a successful user
-sign in by making a request to Okta's Open ID Connect authorization server. See [Get user profile information after sign in](/docs/guides/oie-embedded-sdk-alternate-flows/java/main/#getuserprofileinfo).
+Optionally, you can obtain basic user information after the user is authenticated by making a request to Okta's Open ID Connect authorization server. See [Get user profile information after sign in](/docs/guides/oie-embedded-sdk-alternate-flows/java/main/#getuserprofileinfo).
```
Comment (christinechoy-okta) [inline packages/@okta/vuepress-site/docs/guides/oie-embedded-sdk-use-cases/oie-embedded-sdk-use-case-basic-sign-in/java/integrationsteps.md:88]:
```suggestion
Optionally, you can obtain basic user information after the user is authenticated by making a request to Okta's Open ID Connect authorization server. See [Get user profile information after sign-in](/docs/guides/oie-embedded-sdk-alternate-flows/java/main/#getuserprofileinfo).
```

File: packages/@okta/vuepress-site/docs/guides/oie-embedded-sdk-use-cases/oie-embedded-sdk-use-case-basic-sign-out/java/integrationsteps.md [hunk 4]
```diff
@@ -46,4 +46,4 @@ When the sign out is initiated, create the following flow:
 
 ### Step 3: Send user to the signed out page
 
-After the token is revoked and the session is no longer valid, redirect the user to the desired signed out page.
+After the access token is revoked and the session is no longer valid, redirect the user to the desired signed-out page.
```
Comment (christinechoy-okta) [inline packages/@okta/vuepress-site/docs/guides/oie-embedded-sdk-use-cases/oie-embedded-sdk-use-case-basic-sign-out/java/integrationsteps.md:49]:
```suggestion
After the access token is revoked and the session is no longer valid, redirect the user to the signed-out page.
```

File: packages/@okta/vuepress-site/docs/guides/oie-embedded-widget-use-cases/oie-embedded-widget-use-case-basic-sign-in/java/summaryofsteps.md [hunk 5]
```diff
@@ -1,6 +1,6 @@
 ## Summary of steps
 
-> **Note:** The Okta Java SDK uses the Spring Boot framework to handle OAuth authentication flow. See [Spring Boot Getting Started](https://spring.io/guides/gs/spring-boot/) for more information on Spring.
+> **Note:** The Okta Java SDK uses the Spring Boot framework to handle OAuth 2.0 authentication flow. See [Spring Boot Getting Started](https://spring.io/guides/gs/spring-boot/) for more information on Spring.
```
Comment (christinechoy-okta) [inline packages/@okta/vuepress-site/docs/guides/oie-embedded-widget-use-cases/oie-embedded-widget-use-case-basic-sign-in/java/summaryofsteps.md:3]:
```suggestion
> **Note:** The Okta Java SDK uses the Spring Boot framework to handle the OAuth 2.0 authentication flow. See [Spring Boot Getting Started](https://spring.io/guides/gs/spring-boot/).
```

File: packages/@okta/vuepress-site/docs/guides/oie-embedded-widget-use-cases/oie-embedded-widget-use-case-sign-in-soc-idp/java/summaryofsteps.md [hunk 6]
```diff
@@ -1,6 +1,6 @@
 ## Summary of steps
 
-> **Note:** The Okta Java SDK uses the Spring Boot framework to handle OAuth authentication flow. See [Spring Boot Getting Started](https://spring.io/guides/gs/spring-boot/) for more information on Spring.
+> **Note:** The Okta Java SDK uses the Spring Boot framework to handle OAuth 2.0 authentication flow. See [Spring Boot Getting Started](https://spring.io/guides/gs/spring-boot/) for more information on Spring.
```
Comment (christinechoy-okta) [inline packages/@okta/vuepress-site/docs/guides/oie-embedded-widget-use-cases/oie-embedded-widget-use-case-sign-in-soc-idp/java/summaryofsteps.md:3]:
```suggestion
> **Note:** The Okta Java SDK uses the Spring Boot framework to handle the OAuth 2.0 authentication flow. See [Spring Boot Getting Started](https://spring.io/guides/gs/spring-boot/).
```

**Review by christinechoy-okta — COMMENTED** [review]
Just a few nits.

File: packages/@okta/vuepress-site/docs/guides/oie-embedded-sdk-use-cases/oie-embedded-sdk-use-case-basic-sign-in/java/integrationsteps.md (hunk 3 above)
Comment (vanngo-okta) [reply packages/@okta/vuepress-site/docs/guides/oie-embedded-sdk-use-cases/oie-embedded-sdk-use-case-basic-sign-in/java/integrationsteps.md:88]: leaving as-is (not an adjective)

**Pull request merged**